<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Login successefully executed]]>
    </line>
    <line>
      <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg>Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg><img src=Mon_Feb_04_03_41_29_EST_2019.jpg height=200 width=200></img></a>]]>
    </line>
    <line>
      <![CDATA[Login successefully executed]]>
    </line>
    <line>
      <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg>Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg><img src=Mon_Feb_04_03_41_29_EST_2019.jpg height=200 width=200></img></a>]]>
    </line>
  </reporter-output>
  <suite name="Data Driven Basics" duration-ms="4363" started-at="2019-02-04T08:41:25Z" finished-at="2019-02-04T08:41:29Z">
    <groups>
    </groups>
    <test name="Bank Manager Login Test" duration-ms="3559" started-at="2019-02-04T08:41:25Z" finished-at="2019-02-04T08:41:29Z">
      <class name="com.w2a.testcases.BankManagerLoginTest">
        <test-method status="PASS" signature="loginAsBankManager()[pri:0, instance:com.w2a.testcases.BankManagerLoginTest@26275bef]" name="loginAsBankManager" duration-ms="3383" started-at="2019-02-04T03:41:25Z" finished-at="2019-02-04T03:41:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
      </class> <!-- com.w2a.testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager Login Test -->
    <test name="Add Customer Test" duration-ms="796" started-at="2019-02-04T08:41:29Z" finished-at="2019-02-04T08:41:29Z">
      <class name="com.w2a.testcases.AddCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.w2a.testcases.AddCustomerTest@c333c60]" name="setUp" is-config="true" duration-ms="3208" started-at="2019-02-04T03:41:22Z" finished-at="2019-02-04T03:41:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="addCustomerTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.w2a.testcases.AddCustomerTest@c333c60]" name="addCustomerTest" duration-ms="0" started-at="2019-02-04T03:41:29Z" data-provider="dp" finished-at="2019-02-04T03:41:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mohammed]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[sahan]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[10462.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failing Test Case]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failing Test Case
	at org.testng.Assert.fail(Assert.java:93)
	at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Login successefully executed]]>
            </line>
            <line>
              <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg>Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg><img src=Mon_Feb_04_03_41_29_EST_2019.jpg height=200 width=200></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
        <test-method status="FAIL" signature="addCustomerTest(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.w2a.testcases.AddCustomerTest@c333c60]" name="addCustomerTest" duration-ms="0" started-at="2019-02-04T03:41:29Z" data-provider="dp" finished-at="2019-02-04T03:41:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[farah]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[rashid]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[10463.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failing Test Case]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failing Test Case
	at org.testng.Assert.fail(Assert.java:93)
	at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Login successefully executed]]>
            </line>
            <line>
              <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg>Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target= "_blank" href = Mon_Feb_04_03_41_29_EST_2019.jpg><img src=Mon_Feb_04_03_41_29_EST_2019.jpg height=200 width=200></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.w2a.testcases.AddCustomerTest@c333c60]" name="tearDown" is-config="true" duration-ms="3102" started-at="2019-02-04T03:41:29Z" finished-at="2019-02-04T03:41:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.w2a.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
